src/Package/Impl/Options/R/Editor/RLintOptionsDialog.cs

```diff
@@ -0,0 +1,250 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel;
+using Microsoft.Common.Core.Shell;
+using Microsoft.R.Editor;
+using Microsoft.VisualStudio.R.Package.Options.Attributes;
+using Microsoft.VisualStudio.R.Package.Shell;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.R.Package.Options.R.Editor {
+    public class RLintOptionsDialog: DialogPage {
+        private readonly IWritableREditorSettings _settings;
+
+        public RLintOptionsDialog() {
+            SettingsRegistryPath = @"UserSettings\R_Lint";
+            _settings = VsAppShell.Current.GetService<IWritableREditorSettings>();
```

Comment:
Can we reference `VsAppShell.Current.GetService<IWritableREditorSettings>().LintOptions` directly?